src/index.js

```diff
@@ -55,17 +64,19 @@ const healthCheck = (dispatch, getState, isInitialHealhtcheck = false) => {
     })
     .catch((error) => {
       dispatch(networkReducer.actions.disconnected());
-      // errorHandler.handle(error);
+      errorHandler.handle(error);
     });
 };
 
 const bootstrap = async () => {
   const { dispatch, getState } = store;
+  timeSpan.init(getState);
```

Comment:
Reading this .. rings no bell what's happening in here. Initializing timeSpan? What does that mean?

Reply:
IMO when you read calls  of functions and etc. you must see what's happening in a clear way and just go to the implementation to check how it's done.

i.e. `two.add(four)` is pretty  clear , right? I see `.init` but what does that mean in terms of timeSpan and conversion :S

Reply:
i pass the getState function here since timeSpan needs access to he store.

Reply:
i don't understand how this is related to the issue I've described. IMO timeSpan.init is not good. NO idea what it does.

Reply:
So we will rework this to pass required context as first argument

Reply:
this will be done in separate commit